lib/server.js

```diff
@@ -84,11 +83,10 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
 
   app.post(
     routes.gossip, brRest.when.prefers.ld,
-    validate('continuity-server.gossip'), brRest.linkedDataHandler({
-      // eslint-disable-next-line
-      get: callbackify(async (req, res) => {
-        const localPeerId = config.server.baseUri +
-          '/consensus/continuity2017/voters/' + req.params.voterId;
+    validate('continuity-server.gossip'), asyncHandler(
+      (async (req, res) => {
```

Comment:
This needs some additional style fixes beyond this suggestion:

```suggestion
    validate('continuity-server.gossip'),
    asyncHandler(async (req, res) => {
```